.gitignore

```diff
@@ -127,3 +127,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
```

Comment:
Usually you do not add IDE-specific files to the project-level gitignore. It should contain files related to the language/stack used within the project. 
https://github.com/github/gitignore?tab=readme-ov-file#folder-structure
If you are personally using some tools you can ignore them separately on your machine (e.g. via global ignore https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer)